cypress/e2e/google.cy.js

```diff
@@ -0,0 +1,6 @@
+describe("This is my first cypress test", () => {
+    it("should have a title", () => {
+      cy.visit("https://www.google.com/");
+      cy.title().should("eq", "Google");
+    });
+  });
```

Comment:
End of File

The end of file can be configurated to be added automatically https://docs.microsoft.com/en-us/answers/questions/550115/automatic-format-code-in-visual-studio.html

Reply:
It was solved